.github/workflows/generate-preview.yml

```diff
@@ -28,9 +34,9 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             github.issues.removeLabel({
-              issue_number: github.event.pull_request.number,
```

Comment:
Just a thought here, but what about doing the evaluation syntax here ${{ github.event.pull_request.number }}

Reply:
I'm gonna land what we have, shrug, and hope it works.

if it does then we can iterate. Just want to get something working right now :D